readme.md

```diff
@@ -70,8 +70,26 @@ Known Issues:
 
 4. Open `/wp-admin/admin.php?page=jetpack` in your browser.
 
+#### Production build
+
+The production build will generate minified files without duplicated code (resulting from dependencies) and also will generate the matching sourcemap files.
+
+```
+npm run build-production
+```
+
 #### Development build
 
+The development build will create a build without minifying or deduping code.
+
+```
+npm run build
+```
+
+This will set an environment variable `NODE_ENV=production` before doing a `npm run build`.
```

Comment:
I believe that is supposed to be in the production build section, right? Also, the production build will prepare language files, which will take some time.